Description: use Test::XML for comparing XML instead of 'eq'
 'eq' works most of the time, until you have XML structures which have more
 than one attribute. Their serialization is non-deterministic since the order
 of the attributes is not predictable.
Author: Damyan Ivanov <dmn@debian.org>
Bug: https://rt.cpan.org/Ticket/Display.html?id=74257
Bug-Debian: http://bugs.debian.org/720964
--- a/t/SOAP/WSDL/05_simpleType-list.t
+++ b/t/SOAP/WSDL/05_simpleType-list.t
@@ -36,7 +36,11 @@ ok $xml = $soap->call('test', testAll =>
 #5
 ok ( $xml2 = $soap->call('test', testAll => "1 2" ) , 'Serialized scalar call' );
 #6
-ok( $xml eq $xml2, 'Got expected result');
+SKIP: {
+    skip "Test::XML needed for comparing XML", 1
+        unless eval { require Text::XML };
+    is_xml( $xml, $xml2, 'Got expected result');
+}
 
 #7	
 TODO: {
